src/extension.ts

```diff
@@ -1275,14 +1280,20 @@ async function reportDeleteResult(resourceId: string, shellResult: ShellResult)
     refreshExplorer();
 }
 
-const deleteKubernetes = async (explorerNode?: explorer.ResourceNode) => {
+async function deleteKubernetes(delMode: KubernetesDeleteMode, explorerNode?: explorer.ResourceNode) {
+
+    let delModeString: string = '';
```

Comment:
Minor: suggest `deleteModeArg` rather than `String` to clarify that this is an argument going into `kubectl` (`String` just tells us its type).